### ISSUE TITLE
Publish to pip

### ISSUE THREAD

**Issue by a4vg** [Battleman/zoomdl#42]
**Prerequisite**

- [x] I am asking a question, and not reporting a bug or a feature request
- [x] I have searched the opened and closed issues for a duplicate,

**Question**
Hi, could zoomdl be published to PyPI? This way it can be installed with `pip install zoomdl`.

**Comment by Battleman** [comment]
Sorry for the lack of answer. Yes that should be doable, I'll look into that :)

**Comment by 89z** [comment]
Yes please do. I searched the entire readme, and finally found this issue :(
